src/hb-open-file-private.hh

```diff
@@ -286,6 +286,82 @@ struct TTCHeader
   } u;
 };
 
+/*
+ * Mac Resource Fork
+ */
+
+struct ResourceRefItem
+{
+  HBINT16 id; // Why is not unsigned?
+  HBINT16 nameOffset; // Why is not unsigned?
+  HBUINT8 attr;
+  char dataOffset[3];
+  HBUINT16 reserved;
```

Comment:
Structs are copied from fonttools but I am not sure why signed number were there.